safe/impact_function_v4/test/test_impact_function.py

```diff
@@ -370,6 +372,113 @@ def test_post_processor(self):
                 self.assertIn(field_name, impact_fields)
         print_attribute_table(impact_layer, 1)
 
+    @expectedFailure
```

Comment:
Because it is failed at the moment :p. It doesn't return the correct results. Postprocessors calculation returned the wrong results for analysis layer. I hope @Gustry can help me on this after it is merged.